.github/workflows/tiledb-csharp.yml

```diff
@@ -2,172 +2,154 @@ name: TileDB-CSharp
 
 on:
   push:
-    tags: [ v* ]
+    tags: [ '*' ]
     branches: [ main ]
   pull_request:
     branches: [ main ]
   workflow_dispatch:
 
 jobs:
-  Linux_Test:
-    runs-on: ubuntu-latest
+  Run-Tests:
     strategy:
       matrix:
         # Will be checking following versions
         dotnet: ['5.0']
```

Comment:
Please make an issue to eventually figure out how to run this against both 5.0 and 6.0 -- tried to do that at some point and got stuck with how to get .csproj supported versions spec to accept both.